Documentation/spelling_wordlist.txt

```diff
@@ -173,6 +174,7 @@ balancers
 battlestation
 bcc
 behaviour
+benefitting
```

Comment:
This may also be spelled `benefiting`, both are fine according to https://www.merriam-webster.com/dictionary/benefit

Reply:
We should probably stick to one though, no?

Reply:
Agree. There's no occurrence of either one in `Documentation/` yet before #10203, so I guess we stick to `benefitting`?

Reply:
`benefiting` [looks more common](https://trends.google.com/trends/explore?date=today%205-y&q=benefiting,benefitting), but I don't have a strong opinion either way :-)